app/styles/components/signup-form.scss

```diff
@@ -5,4 +5,25 @@
   input[type="submit"] {
     width: 100%;
   }
+  @include media($lg-screen) {
+    @include shift(2);
+    @include span-columns(8);
+  }
+  @include media($sm-screen) {
+    @include shift(1);
+    @include span-columns(10);
+  }
+  @include media($sm-screen) {
+    @include shift(0);
+    @include span-columns(12);
+    padding: 0 8px;
+  }
```

Comment:
Looks like there is some repetition on the `sm-screen` here.

Also, I see there being some padding on the last `sm-screen`. If we wrap the login in a `.container`, then it should be fine.